jpype/_jcustomizer.py

```diff
@@ -17,7 +17,7 @@
 __all__ = ['JImplementationFor', 'JConversion']
 
 
-def JConversion(cls, exact=None, instanceof=None, attribute=None):
+def JConversion(cls, exact=None, instanceof=None, attribute=None, none=None):
```

Comment:
Cosmetics - I think I would call the attribute to deny certain types "exclude" instead of "none", as in python "none" is potentially confusing with the "None" keyword and type.

Reply:
Does the same thing apply to the user guide 

https://jpype.readthedocs.io/en/latest/userguide.html#type-matching
https://jpype.readthedocs.io/en/latest/userguide.html#java-conversions